Makefile

```diff
@@ -103,12 +103,19 @@ binary: cmd/skopeo
 	${CONTAINER_RUNTIME} run --rm --security-opt label=disable -v $$(pwd):/src/github.com/containers/skopeo \
 		skopeobuildimage make bin/skopeo $(if $(DEBUG),DEBUG=$(DEBUG)) BUILDTAGS='$(BUILDTAGS)'
 
-# Update nix/nixpkgs.json its latest master commit
+# Update nix/nixpkgs.json its latest stable commit
 .PHONY: nixpkgs
 nixpkgs:
 	@nix run -f channel:nixos-20.03 nix-prefetch-git -c nix-prefetch-git \
 		--no-deepClone https://github.com/nixos/nixpkgs > nix/nixpkgs.json
 
+# Build statically linked binary
+.PHONY: static
+static:
```

Comment:
I’m not really a fan of making this easy.

Reply:
Yes I am now going to use this directly; my PR for buildah CI and podman CI are NOT using such `make static` but `nix build -f nix/` directly, see:
- https://github.com/containers/buildah/pull/2428/files#diff-0f7067c31caedf7df2739c9b07efa122R257-R276
https://github.com/containers/podman/pull/6402/files#diff-0f7067c31caedf7df2739c9b07efa122R763-R778

Anyway, at least I am trying to synchronize such support between skopeo/buildah/podman for newbie, also see:
- https://github.com/containers/buildah/pull/2428/files#diff-b67911656ef5d18c4ae36cb6741b7965R50-R56
- https://github.com/containers/podman/pull/6402/files#diff-b67911656ef5d18c4ae36cb6741b7965R229-R234